src/objective-c/tests/ThirdPartyTests/Libuv/LibuvGetHostnameTests.m

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "test/runner.h"
+#import "test/task.h"
+
+// Tests in:
+// libuv/test/test-gethostname.c
+
+TEST_DECLARE(gethostname)
+
+#define TASK_LIST_START_LOCAL static task_entry_t TEST_TASKS[] = {
```

Comment:
I'm not familiar with this test framework, so maybe this is meaningful in some other way, but does this need to be redefined identically in every file? Maybe move the definition to a shared header if not.

Since this is just a static variant on libuv's own `TASK_LIST_START`, consider this instead, which may "just work" if libuv changes the meaning of TASK_LIST_START in backwards-compatible ways.
```
#define TASK_LIST_START_LOCAL static TASK_LIST_START
```

Or maybe just use `static TASK_LIST_START` in these test files and avoid the custom macro altogether.

Reply:
It doesn't work because `TASKS` was declared as `external` in `libuv/test/runner.h`, it needs to be a new name.
I also tried just use `static task_entry_t TEST_TASKS[] = {` directly without defining a macro, which compiles but clang-format will mess up the format.

I can move the macro definition into a common header, not sure if it's worthwhile given it's only one line, maybe we can do it later when we have more code needs to be shared?